src/cmake/thirdparty/SetupAxomThirdParty.cmake

```diff
@@ -27,6 +27,9 @@ if (UMPIRE_DIR)
         message(FATAL_ERROR "Given UMPIRE_DIR is not a directory: ${UMPIRE_DIR}")
     endif()
 
+    # white238 - Workaround to get around umpire's exported camp logic not working
+    set(camp_DIR "${CAMP_DIR}/lib/cmake/camp")
```

Comment:
Is there an issue for this?

Tag: @davidbeckingsale

Reply:
This is not really an issue - we modified our installed CMake configs to handle relocation, but that means we no longer hard code any camp_DIR in the umpire (or RAJA) packages. Best practice is to treat camp as a true separate TPL, which means providing a camp_DIR - you could run find_package(camp) in Axom to have more control, or you can defer the search for camp to the Umpire/RAJA packages (but it still requires a camp_DIR).